src/Actions/Family/CreateFamily.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Choredo\Actions\Family;
+
+use Assert\Assertion;
+use Choredo\Hydrators;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Teapot\StatusCode;
+use Zend\Diactoros\Response\JsonResponse;
+
+class CreateFamily
+{
+    /**
+     * @var Hydrators\Family
+     */
+    private $familyHydrator;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * CreateFamily constructor.
+     * @param Hydrators\Family $familyHydrator
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(Hydrators\Family $familyHydrator, EntityManagerInterface $entityManager)
+    {
+        $this->familyHydrator = $familyHydrator;
+        $this->entityManager = $entityManager;
+    }
+
+    public function __invoke(ServerRequestInterface $request, ResponseInterface $response)
+    {
+        $body = $request->getBody()->getContents();
+        Assertion::isJsonString($body);
```

Comment:
Should we use more descriptive error messages than the Assertion defaults?

Reply:
This is how a typical error renders: 

```
{
    "errors": [
        "The following 1 assertions failed:\n1) Family::weekStartDay: Value \"buttday\" is not an element of the valid values: sunday, monday, tuesday, wednesday, thursday, friday, saturday\n"
    ]
}
```

I kinda like that. Thoughts?

Reply:
Depends on what the error is for. For logging that's great, for showing to the user that's rubbish.